sup3r/pipeline/forward_pass.py

```diff
@@ -737,25 +738,18 @@ def __init__(self, file_paths, model_kwargs, fwp_chunk_shape,
                                             self.spatial_pad,
                                             self.temporal_pad)
 
-        msg = ('The average memory of a forward pass output chunk is '
-               f'{self.output_chunk_mem / 1e9:.2f} GB. Exceeding 1.5 GB could'
-               ' result in constant model output.')
-        if self.output_chunk_mem / 1e9 > 1.5:
-            logger.warning(msg)
-            warnings.warn(msg)
-
         self.preflight()
 
     @property
-    def output_chunk_mem(self):
-        """Get average memory of forward pass output chunk. This is used to
-        check if memory exceeds limit where model can return constant output.
-        Returns memory in bytes, assuming output has float32 type."""
-        n_elements = (self.fwp_chunk_shape[0] + 2 * self.spatial_pad)
-        n_elements *= (self.fwp_chunk_shape[1] + 2 * self.spatial_pad)
-        n_elements *= (self.fwp_chunk_shape[2] + 2 * self.temporal_pad)
-        n_elements *= len(self.output_features)
-        return 4 * n_elements
+    def failed_chunks(self):
+        """Check whether any forward passes have generated constant output."""
+        return self._failed_chunks
+
+    @failed_chunks.setter
+    def failed_chunks(self, failed):
```

Comment:
This property+setter is functionally identical to just having a public self.failed_chunks attribute without the methods, any reason for doing the property+setter?

Reply:
Haha just figured we should make it clear there is this dedicated thing to check for failures rather than adding to the big list of attrs in init.

Reply:
Gotcha, sounds good!